runtime/src/main/java/org/corfudb/runtime/collections/PersistedStreamingMap.java

```diff
@@ -40,6 +45,37 @@
         RocksDB.loadLibrary();
     }
 
+    /**
+     * A set of options defined for disk-backed {@link CorfuTable}.
+     *
+     * For a set of options that dictate RocksDB memory usage can be found here:
+     * https://github.com/facebook/rocksdb/wiki/Memory-usage-in-RocksDB
+     *
+     * Block Cache:  Which can be set via Options::setTableFormatConfig.
+     *               Out of box, RocksDB will use LRU-based block cache
+     *               implementation with 8MB capacity.
+     * Index/Filter: Is a function of the block cache. Generally it infates
+     *               the block cache by about 50%. The exact number can be
+     *               retrieved via "rocksdb.estimate-table-readers-mem"
+     *               property.
+     * Write Buffer: Also known as memtable is defined by the ColumnFamilyOptions
+     *               option. The default is 64 MB.
+     */
+    public static Options getPersistedStreamingMapOptions() {
```

Comment:
Can you make all these parameters configurable? 
At least pass `Options` as a parameter in the constructor?
Or any other way that helps make all the parameters configurable

Reply:
Ok.

Reply:
Its kind of messy, because its passed to the constructor through a supplier. CorfuTable can be refactored in a different patch.